.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Regression
+about: Report a regression in Rust.
+labels: C-bug regression-untriaged
+---
+<!--
+Thank you for filing a regression report! 🐛 Please provide a short summary of the regression,
+along with any information you feel is relevant to replicate it.
```

Comment:
I like what LeSeulArtichaut suggested to explain what a regression is. Therefore I'd clarify a little bit here:

```suggestion
Thank you for filing a regression! 🐛  A regression is an error in a component that used to work but doesn't anymore.

Please provide a short summary of the regression, along with any information you feel is relevant to replicate it.
```

Reply:
Hmm, not *all* regressions are new errors that didn't used to occur. What do you think of this?

```suggestion
Thank you for filing a regression report! 🐛 A regression is something that changed between versions of Rust but was not supposed to.

Please provide a short summary of the regression,
along with any information you feel is relevant to replicate it.
```

Reply:
Probably both more correct than "my" definition and clearer in layman's terms. :+1: